### PR TITLE
Added module-like Python scripts which could come handy to somebody

### PR DESCRIPTION
**Pull request by real-zephex**
**Downloads Sorter**
ℹ️ This commit adds a Python script that organizes files in the downloads folder based on their file types. The script creates folders for different file types, moving corresponding files accordingly. Additionally, it includes a function to move subfolders into a separate "Folders" folder.

**Currency Converter**
🌐 Introducing a currency converter program! This commit adds a versatile Python script designed to be used as a module. Utilizing the forex--python module, it provides exchange rate functionality. Developers can easily modify it according to specific needs.

**Weather Forecast**
🌦️ This commit introduces a Python script for retrieving weather information about a city using the OpenWeatherMap API. While designed as a module, it's adaptable for custom modifications. API key is required.

**Quotes Fetcher**
📜 Adding a Python script to fetch quotes from the zenquotes API! This script can generate random quotes, the quote of the day, or a specific number of quotes (up to 50). No API key required, making it easy to integrate.

🚀 Ready for merge!

### CODE REVIEW

**Comment by real-zephex** [comment]
*Added currency.md

**Review by yashkite — CHANGES_REQUESTED** [review]
some changes are requested

**Review by real-zephex — COMMENTED** [review]
since we have declared it a class method, it is bound to the class instead of class instance. The first parameter refers to the class itself.

File: Small Projects/Python/Quotes/quote.py
[the remark this answers is not in the document]
Comment (real-zephex) [reply Small Projects/Python/Quotes/quote.py:1]: doing it rn

[the remark this answers is not in the document]
Comment (real-zephex) [reply Small Projects/Python/Quotes/quote.py:1]: done

**Comment by real-zephex** [comment]
@yashkite 
I made the required changes, please review them!

**Review by yashkite — APPROVED** [review]
Thanks for the PR.

**Pull request merged**